src/components/create-guild/ChooseTemplate/components/TemplateCard.tsx

```diff
@@ -241,6 +243,8 @@ const TemplateCard = ({
         >
           {selected ? (
             <Circle
+              id={`role-checkbox`}
```

Comment:
Do we actually need the ID attribute here? (I think the `data-test` might be sufficient, but maybe I miss something)